src/styles/stars.scss

```
@@ -0,0 +1,30 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url("../images/star.svg") center no-repeat;
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+}
+
+.stars--1 :nth-child(-n+1) {
+  background-image: url("../images/star-active.svg");
+}
+
+.stars--2 :nth-child(-n+2) {
+  background-image: url("../images/star-active.svg");
+}
+
+.stars--3 :nth-child(-n+3) {
+  background-image: url("../images/star-active.svg");
+}
+
+.stars--4 :nth-child(-n+4) {
+  background-image: url("../images/star-active.svg");
+}
+
+.stars--5 :nth-child(-n+5) {
+  background-image: url("../images/star-active.svg");
+}
```

```suggestion
.stars--1 :nth-child(-n+1) {
  background-image: url("../images/star-active.svg");
}

.stars--2 :nth-child(-n+2) {
  background-image: url("../images/star-active.svg");
}

.stars--3 :nth-child(-n+3) {
  background-image: url("../images/star-active.svg");
}

.stars--4 :nth-child(-n+4) {
  background-image: url("../images/star-active.svg");
}

.stars--5 :nth-child(-n+5) {
  background-image: url("../images/star-active.svg");
}
```

combine these selectors with coma